sdk/vision/azure-ai-vision-imageanalysis/azure/ai/vision/imageanalysis/_patch.py

```diff
@@ -0,0 +1,20 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+"""Customize generated code here.
+
+Follow our quickstart for examples: https://aka.ms/azsdk/python/dpcodegen/python/customize
+"""
+from typing import List
+
+__all__: List[str] = []  # Add all objects you want publicly available to users at this package level
+
+
+def patch_sdk():
+    """Do not remove from this file.
+
+    `patch_sdk` is a last resort escape hatch that allows you to do customizations
+    you can't accomplish using the techniques described in
+    https://aka.ms/azsdk/python/dpcodegen/python/customize
+    """
```

Comment:
nit: please follow https://aka.ms/azsdk/python/dpcodegen/python/customize to write customized code, otherwise your manual change will be overwritten when next generation.

Reply:
Thank you @msyyc, I'll have a look at this first thing in the morning.